lib/compat/wordpress-6.8/site-editor.php

```diff
@@ -136,14 +136,45 @@ function gutenberg_styles_wp_die_handler( $default_handler ) {
 }
 add_filter( 'wp_die_handler', 'gutenberg_styles_wp_die_handler' );
 
+/**
+ * Registers the stylebook theme support.
+ * Ensures that the stylebook theme support is available in the editors by setting show_in_rest to true.
+ * Backports into wp-includes\theme.php create_initial_theme_features().
+ */
+add_action(
+	'setup_theme',
+	function () {
+		register_theme_feature(
+			'stylebook',
+			array(
+				'description'  => __( 'test' ),
```

Comment:
of course, this isnt supposed to say test  😆 I will fix it when I get back to work next week.